src/makeCalculator.js

```diff
@@ -37,7 +37,55 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, value) {
+      callback(value);
+
+      return this;
+    },
+
+    get add() {
+      return (value) => {
+        this.result += value;
+
+        return this.result;
```

Comment:
we pass our data directly to our method, we don't need to create a closure here for this